django/contrib/admin/static/admin/css/widgets.css

```diff
@@ -49,6 +49,7 @@
     padding: 8px;
 }
 
+.tabular .selector-chosen-title label,
```

Comment:
This visual regression was introduced in a0f50c2a483678d31bd1ad6f08fd3a0b8399e27b
We might want to do something like:
```diff
--- a/django/contrib/admin/static/admin/css/widgets.css
+++ b/django/contrib/admin/static/admin/css/widgets.css
@@ -49,8 +49,7 @@
     padding: 8px;
 }
 
-.tabular .selector-chosen-title label,
-.aligned .selector-chosen-title label {
+.selector-chosen-title label {
     color: var(--header-link-color);
     width: 100%;
 }
@@ -61,7 +60,7 @@
     padding: 8px;
 }
 
-.aligned .selector-available-title label {
+.selector-available-title label {
     width: 100%;
 }
```